protocol/cmd/dydxprotocold/cmd/root.go

```diff
@@ -90,7 +91,7 @@ func NewRootCmdWithInterceptors(
 		dbm.NewMemDB(),
 		nil,
 		true,
-		viper.New(),
```

Comment:
Can this be replaced with:
```
initAppOptions := viper.New()
initAppOptions.Set(flags.FlagHome, tempDir())
tempApp(..., initAppOptions)
```

Was hoping to keep away from test utilities in Cosmos SDK as part of the production code.